tests/test_sqla_yaml_fixtures.py

```diff
@@ -499,3 +526,29 @@ def test_custom_loader(session):
     assert users[0].email == 'deedee@ramones.org'
     assert users[1].username == 'joey'
     assert users[1].email == 'joey@ramones.org'
+
+
+def test_many_2_many(session):
+    fixture = """
+- User:
+  - __key__: joey
+    username: joey
+    email: joey@example.com
+  - __key__: bobby
+    username: bobby
```

Comment:
oh. and there is no "Bobby" Ramone :grin: https://en.wikipedia.org/wiki/Ramones#Band_members

Reply:
haha - I just realized the ramones reference